lib/components/ms-modal/utils.ts

```diff
@@ -57,9 +56,16 @@ export async function askQuestion(title: Translatable, subtitle: Translatable, o
 }
 
 export async function getPasswordFromUser(options: GetPasswordOptions): Promise<string | null> {
+  const { isLargeDisplay } = useWindowSize();
```

Comment:
Move this to the top of the file as a global variable, same remark for `getTextFromUser`